ogre_mesh_viewer.py

```diff
@@ -528,6 +537,14 @@ def _toggle_axes(self):
         
         self.axes_visible = not self.axes_visible
 
+    def _toggle_grid(self):
+        self.grid_visible = not self.grid_visible
+
+        if self.grid_visible is True:
```

Comment:
```suggestion
        if self.grid_visible:
```
I find the `is True` overly verbose. Please remove here and below